docs/command_line_tools.md

```diff
@@ -207,8 +209,8 @@ Generated project structure
 
 Your application code and tests will be located in the following project directories:
 
-- `./src/main/java` - Generated Java application code stub
```

Comment:
Not sure we need to remove this and the test/java line below. They are still valid when generating a Java based project, no?

Reply:
yeah I will change it to kotlin

Reply:
?

No, keep the old + the new you already added